style.css

```diff
@@ -120,7 +120,6 @@ height: 500px;
 }
 
 .locations {
-position: absolute;
```

Comment:
@fessjarmer we can't position elements with `flex` if they're positioned `absolute` 🙃 

More on that👇  
https://chenhuijing.com/blog/flexbox-and-absolute-positioning/#%F0%9F%9A%B2